serverless/serverless
# Changed CloudFormation files to yaml instead of json

## What did you implement:

**_Implementing Issue:**_ #2235


## How did you implement it:

Dump to yaml instead of json when saving. Keep the internal structure javascript objects. Changed the filenames to yaml for the create/update stack and the cloudformation template.


## How can we verify it:

Deploy a project and see that yaml files are generated in the .serverless folder instead of json files.
Verify that the yaml works in CloudFormation.


## Todos:

**_Is this ready for review?:**_ YES
